### PR TITLE
STIJ-246: Added fixes for inconsistent components.

### PR DESCRIPTION
**Pull request by meirege**
## Description
Added fixes for links in components + remove underline from links that use the button styling.

## Motivation and Context
This functionality was broken due to merge conflicts in the 2.8.0 branch. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the CHANGELOG accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: components/41-organisms/footer/_footer.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // General footer styling.
 footer.footer {
+  @include no-link-style;
```
Comment (delrueba) [inline components/41-organisms/footer/_footer.scss]: Doesn't match stad.gent styling.
Also, I'd limit usage of this mixin to molecules.

File: components/11-base/base/_base.scss [hunk 2]
```diff
@@ -83,6 +83,12 @@ img {
   clip: rect(0 0 0 0);
 }
 
+
+%no-link-style,
+.no-link-style {
```
Comment (delrueba) [inline components/11-base/base/_base.scss]: I like that there's also a class available.

File: components/41-organisms/footer/_footer.scss (hunk 1 above)
Comment (meirege) [reply components/41-organisms/footer/_footer.scss]: Fixed.

**Comment by meirege** [comment]
@delrueba Actually no, the header still needs it's links to be underlined when focussed or hovered.
The no-link-style mixin is meant to remove underlining for certain elements like buttons and links, so it's not really a correct use here.

**Pull request merged**